src/server/auth/cmds/cmds.go

```diff
@@ -681,6 +681,36 @@ func GetEnterpriseRoleBindingCmd() *cobra.Command {
 	return cmdutil.CreateAlias(get, "auth get enterprise")
 }
 
+// RotateAuthToken returns a cobra command that rotates an auth token for a User
+func RotateAuthToken() *cobra.Command {
+	var pachToken string
+	rotateToken := &cobra.Command{
+		Use:   "{{alias}}",
+		Short: "Rotate the auth token for a user",
+		Long:  "Rotate the auth token for a user",
+		Run: cmdutil.RunBoundedArgs(0, 0, func(args []string) error {
+			c, err := newClient(false)
+			if err != nil {
+				return errors.Wrapf(err, "could not connect")
+			}
+			defer c.Close()
+
+			req := &auth.RotateAuthTokenRequest{
+				PachToken: pachToken,
+			}
+			resp, err := c.RotateAuthToken(c.Ctx(), req)
+			if err != nil {
+				return err
+			}
+			fmt.Printf("Pachyderm auth token:\n%s\n", resp.PachToken)
```

Comment:
We should make this explicit that it's the root token, with the same caveats as `Activate` that the old root token will be revoked and this is the only way to authenticate if auth is broken.